best-practices/restrict_image_registries/restrict_image_registries.yaml

```diff
@@ -8,7 +8,8 @@ metadata:
       Images from unknown registries may not be scanned and secured. 
       Requiring use of known registries helps reduce threat exposure.
 spec:
-  validationFailureAction: audit
+  background: false
+  validationFailureAction: enforce
```

Comment:
any reason why this was changed to enforce? For most of the sample policies, we keep as audit and allow the user to set as enforce if they wish.

Reply:
Not any specific reason. I was testing using enforce. I will revert back to audit.